### PR TITLE
Hot-fix - css fixes, menu bounce

### PR DESCRIPTION
**Pull request by randyhoulahan**

### CODE REVIEW

File: app/template.js [hunk 1]
```diff
@@ -147,4 +140,16 @@ define(['app', 'angular','text!./toast.html',
         }
 
     }]);
+
+    function loadIfPhoenix(){
+        if(!document.getElementById("phoenix")) return
```
Comment (sbilodeau) [inline app/template.js]: I think is better if you test that the <body> element is phoenix. the use if id may have create interference

Comment (randyhoulahan) [reply app/template.js]: Better?

File: app/template.js [hunk 2]
```diff
@@ -2,14 +2,7 @@ define(['app', 'angular','text!./toast.html',
 'lodash', 'PageHeaderFixed','PageHeader','PageFooter','ngVue','providers/realm','./directives/bread-crumbs'], function(app, ng, toastTemplate, _, PageHeaderFixedComp, PageHeaderComp, PageFooterComp) {
     'use strict';
```
Comment (blaisefonseca) [inline app/template.js]: you are still loading the component and files before making decision to loadIfPhoenix

File: app/template.js [hunk 3]
```diff
@@ -147,4 +140,18 @@ define(['app', 'angular','text!./toast.html',
         }
 
     }]);
+
+    function loadIfPhoenix(){
+        var phoenixElm =  document.getElementById("phoenix")
```
Comment (blaisefonseca) [inline app/template.js]: use server side variable to make decision passed from header which is much better check line 73 server.js
if(req.headers['x-wpt']=='phoenix')

Comment (sbilodeau) [reply app/template.js]: No need it’s just temporary until we lauch. After we will not go to the old template so it will be default

Comment (sbilodeau) [reply app/template.js]: No need it’s just temporary until we lauch. After we will not go to the old template so it will be default

**Pull request merged**